cc/logging.cc

```diff
@@ -24,15 +24,40 @@
 
 namespace minigo {
 namespace internal {
+
 namespace {
+
 absl::Mutex* mutex() {
   static auto* m = new absl::Mutex();
   return m;
 }
+
+void DumpStackTrace(std::ostream* os) {
```

Comment:
potentially stupid question: can you pass std::ostream& os and avoid (*os)?

Reply:
I could, but we try and follow the google style guide, which only allows for const reference parameter.